src/py-opentimelineio/opentimelineio/core/__init__.py

```diff
@@ -22,9 +22,12 @@
     instance_from_schema,
     register_serializable_object_type,
     register_upgrade_function,
+    register_downgrade_function,
     set_type_record,
     _serialize_json_to_string,
     _serialize_json_to_file,
+    type_version_map,
+    release_to_schema_version_map,
```

Comment:
Should this one be exposed in `__all__`?

Reply:
good catch!